src/govuk/components/pagination/pagination.yaml

```
@@ -0,0 +1,243 @@
+params:
+- name: previous
+  type: object
+  required: false
+  description: A link to the previous page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The link text to the previous page. Defaults to 'Previous page', where 'page' is visually hidden.
+  - name: visuallyHiddenText
+    type: string
+    required: false
+    description: The visually hidden label for the pagination item which will be applied to an `aria-label` and announced by screen readers on the previous page link.
+  - name: labelText
+    type: string
+    required: false
+    description: Optional label that goes underneath the previous page's link, providing further context for the user about where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The previous page's URL.
+  - name: attributes
+    type: object
+    required: false
+    description: HTML attributes (for example data attributes) to add to the anchor.
+- name: next
+  type: object
+  required: false
+  description: A link to the next page, if there is one.
+  params:
+  - name: text
+    type: string
+    required: false
+    description: The link text to the next page. Defaults to 'Next page', where 'page' is visually hidden.
+  - name: visuallyHiddenText
+    type: string
+    required: false
+    description: The visually hidden label for the pagination item which will be applied to an `aria-label` and announced by screen readers on the next page link.
+  - name: labelText
+    type: string
+    required: false
+    description: Optional label that goes underneath the next page link providing further context for where it goes.
+  - name: href
+    type: string
+    required: true
+    description: The next page URL.
+  - name: attributes
+    type: object
+    required: false
+    description: HTML attributes (for example data attributes) to add to the anchor.
+- name: landmarkLabel
+  type: string
+  required: false
+  description: The label for the navigation landmark that wraps the pagination. Defaults to "results".
+- name: items
+  type: array
+  required: false
+  description: Array of link objects.
+  params:
+  - name: number
+    type: string
+    required: false
+    description: The pagination item text - usually a page number.
+  - name: visuallyHiddenText
+    type: string
+    required: false
+    description: The visually hidden label for the pagination item which will be applied to an `aria-label` and announced by screen readers on the pagination item link.
```

```suggestion
    description: The visually hidden label (for the pagination item) which will be applied to an `aria-label` and announced by screen readers on the pagination item link.
```
In those last few words, are we referring to a link ON the pagination item item, or a link to the pagination item. If it's the latter, could change "pagination item link" to "link to the pagination item".

---

It's the former. We're not referring to the target of the link here, we're referring to the link html element itself.